### ISSUE TITLE
Probably a typo in calculating data_precision?

### ISSUE THREAD

**Issue by wbcpeter3000** [philipperemy/lead-lag#16]
Hi，

In file lead_lag.py, from line 45 to 48,

```py
data_precision = min(
    min([(ts2.index[i] - ts2.index[i - 1]).total_seconds() for i in range(1, len(ts2))]),
    min([(ts2.index[i] - ts2.index[i - 1]).total_seconds() for i in range(1, len(ts2))])
)
```

Would it be:

```py
data_precision = min(
    min([(ts1.index[i] - ts1.index[i - 1]).total_seconds() for i in range(1, len(ts1))]),  # replace ts2 with ts1
    min([(ts2.index[i] - ts2.index[i - 1]).total_seconds() for i in range(1, len(ts2))])
)
```

Best,

**Comment by philipperemy** [comment]
@wbcpeter3000 thanks for pointing out. I will fix it ;)

**Comment by philipperemy** [comment]
Fixed in 2.2.
